Typescript declaration file contains syntax errors.

After updating the api version to 2.1.1 I got these errors:

```
13     constructor(options: Rollbar.Configuration?);
                                                 ~
node_modules/rollbar/index.d.ts(13,47): error TS1005: ',' expected.

18     public lastError(): Error?;
                                ~
node_modules/rollbar/index.d.ts(18,30): error TS1144: '{' or ';' expected.

43     export type Callback = (err: Error?, response: object) => void;
                                         ~
node_modules/rollbar/index.d.ts(43,39): error TS1005: ',' expected.

43     export type Callback = (err: Error?, response: object) => void;
                                          ~
node_modules/rollbar/index.d.ts(43,40): error TS1138: Parameter declaration expected.
``` 
Probably the idea was to make Error an optional value

--- a/index.d.ts
+++ b/index.d.ts
@@ -15,7 +15,7 @@ declare class Rollbar {
 
     public global(options: Rollbar.Configuration): Rollbar;
     public configure(options: Rollbar.Configuration): Rollbar;
-    public lastError(): MaybeError;
+    public lastError(): Rollbar.MaybeError;
 
     public log(...args: Rollbar.LogArgument[]): Rollbar.LogResult;
     public debug(...args: Rollbar.LogArgument[]): Rollbar.LogResult;

--- a/index.d.ts
+++ b/index.d.ts
@@ -10,12 +10,12 @@
 export = Rollbar;
 
 declare class Rollbar {
-    constructor(options: Rollbar.Configuration?);
+    constructor(options?: Rollbar.Configuration);
     static init(options: Rollbar.Configuration): Rollbar;
 
     public global(options: Rollbar.Configuration): Rollbar;
     public configure(options: Rollbar.Configuration): Rollbar;
-    public lastError(): Error?;
+    public lastError(): MaybeError;
 
     public log(...args: Rollbar.LogArgument[]): Rollbar.LogResult;
     public debug(...args: Rollbar.LogArgument[]): Rollbar.LogResult;
@@ -27,6 +27,7 @@ declare class Rollbar {
 }
 
 declare namespace Rollbar {
+    export type MaybeError = Error | undefined | null;
     export type Level = "debug" | "info" | "warning" | "error" | "critical";
     export interface Configuration {
         version?: string;
@@ -39,8 +40,9 @@ declare namespace Rollbar {
         enabled?: boolean;
         captureUncaught?: boolean;
         captureUnhandledRejections?: boolean;
+        payload?: object;
     }
-    export type Callback = (err: Error?, response: object) => void;
+    export type Callback = (err: MaybeError, response: object) => void;
     export type LogArgument = string | Error | object | Callback | Date | any[];
     export interface LogResult {
         uuid: string;

